client/multi-currency/enabled-currencies-list/style.scss

```
@@ -124,9 +124,7 @@
 		flex: 1;
 		display: flex;
 		flex-direction: column;
-		padding-right: 16px;
-		padding-left: 5px;
-		margin-left: -5px;
+		margin: 0 -24px -24px 0;
```

Maybe use the constants here, so that on upgrade it won't break?
```suggestion
		margin: 0 -#{$grid-unit-30} -#{$grid-unit-30} 0;
```

---

Thanks @frosso I forgot to use the var! Addressed in 9b71716